events.yaml

```diff
@@ -0,0 +1,35 @@
+x-events:
+  - name: cancel_install_failed_start
+    id: "CIE00001"
```

Comment:
What's the distinction between `CIE*` and `GIF*`?

Reply:
thes are just some dumb convention I used. CIE - Cluster Install Error GIF - Generate Image Failure
Of course these are just examples we can use whatever convention we want for the IDs

Reply:
why we need id? sounds like name is enough no?